description/description.rdfs.ttl

```diff
@@ -156,4 +174,59 @@ haDes:numberOfReels a rdf:Property ;
     skos:definition "Het aantal filmrollen/-spoelen (ongeacht het type) waaruit een gedigitaliseerde film bestaat."@nl;
     skos:definition "The number of film reels (regardless of type) that make up a digitized film."@en;
     skos:definition "Le nombre de bobines de film (quel que soit le type) qui composent un film numérisé."@fr;
-    rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
\ No newline at end of file
+    rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
+
+haDes:iec60094Type a rdf:Property ;
+  rdfs:domain haObj:PhysicalCarrier ;
+  rdfs:label "iec-60094 type"@en ;
+  rdfs:label "iec-60094 type"@nl ;
+  rdfs:label "iec-60094 type"@fr ;
+  skos:definition "Het iec-60094 type van de compact cassette die duidt op het magnetisch materiaal dat als bindmiddel werd gebruikt bovenop de polyester onderlaag van de cassette."@nl ;
+  skos:definition "The IEC-60094 Type of the compact cassette that indicates the magnetic material used as a binder on top of the polyester bottom layer of the cassette."@en ;
+  skos:definition "Le type IEC-60094 de la cassette compacte qui indique le matériau magnétique utilisé comme liant sur le dessus de la couche inférieure en polyester de la cassette."@fr ;
+  rdfs:range skos:Concept ;
+  rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
+
+haDes:audioRecordingSpeed a rdf:Property ;
+  rdfs:domain haObj:PhysicalCarrier ;
+  rdfs:label "audio recording speed"@en ;
+  rdfs:label "audio opnamesnelheid"@nl ;
+  rdfs:label "vitesse d'enregistrement audio"@fr ;
+  skos:definition "De snelheid waarmee openspoelaudio of een grammofoonplaat wordt opgenomen."@nl ;
+  skos:definition "The speed with which open reel audio or a gramophone record is recorded."@en ;
+  skos:definition "La vitesse à laquelle l'audio ouvert ou un enregistrement gramophone est enregistré."@fr ;
+  rdfs:range skos:Concept ;
+  rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> .
+
+haDes:audioNoiseReduction a rdf:Property ;
+  rdfs:domain haObj:PhysicalCarrier ;
+  rdfs:label "audio noise reduction"@en ;
+  rdfs:label "audio ruisonderdrukking"@nl ;
+  rdfs:label "réduction du bruit audio"@fr ;
+  skos:definition "De techniek die gebruikt wordt voor ruisonderdrukking bij audio."@nl;
+  skos:definition "La technologie utilisée pour la réduction du bruit de l'audio."@fr;
+  skos:definition "The technology used for noise reduction in audio."@en;
+  rdfs:range skos:Concept ;
+  rdfs:isDefinedBy <https://data.hetarchief.be/ns/description/> . 
+
+haDes:numberOfAudioTracks a rdf:Property ;
+  rdfs:domain haObj:CarrierRepresentation ;
+  rdfs:label "number of audio tracks"@en ;
+  rdfs:label "aantal audiosporen"@nl ;
+  rdfs:label "nombre de pistes audio"@fr ;
+  skos:definition "Het aantal parallele audio sporen die de representatie bevat."@nl;
```

Comment:
Is de notie 'parallel' belangrijk hier?

Reply:
ik ook niet :) Maar ik denk het niet nee :)